iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/auth/entity/Role.java

```diff
@@ -36,62 +37,196 @@
 public class Role {
 
   private String name;
-  private List<PathPrivilege> privilegeList;
+  private List<PathPrivilege> pathPrivilegeList;
+
+  private Set<Integer> sysPrivilegeSet;
+
+  private Set<Integer> sysPriGrantOpt;
+
+  public static final int sysPriSize = 10;
```

Comment:
```suggestion
  public static final int SYS_PRI_SIZE = 10;
```

Reply:
Can we directly use Enum to init this field? Because in future, other  people add a new auth, he may forget to change here.